#This message alows for artificialinteligence_node to send AI data to control_node
uint8 state
